src/sentry/models/identity.py

```diff
@@ -47,16 +50,15 @@ def get(cls, type, instance):
 
 class Identity(Model):
     """
-    A unique identity with an external provider (e.g. GitHub).
+    A verified link between a user and a third party identity.
     """
     __core__ = False
 
     idp = FlexibleForeignKey('sentry.IdentityProvider')
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
This needs to be nullable.

Reply:
[Addressed in this comment](https://github.com/getsentry/sentry/pull/6708#issuecomment-354855601)